operator/roles/kiali-deploy/tasks/openshift/os-main.yml

```diff
@@ -65,5 +65,9 @@
 - name: Process OpenShift Console Links
   k8s:
     definition: "{{ lookup('template', 'templates/openshift/console-links.yaml') }}"
+  vars:
+    # For some reason, in some cases when the operator runs, the kiali.io/member-of tag is not set yet.
```

Comment:
Ah.. yes. That is because those labels are only ever added when accessible_namespaces is NOT `**` as you see here:

https://github.com/kiali/kiali/blob/v1.12/operator/roles/kiali-deploy/tasks/main.yml#L400-L405

So those are not good to look for since you need something when it is `**`

So I would take out the comment "for some reason in some cases" -- its in all cases when the operator runs with accessible namespaces of ** it does not create those labels because, up to now, those labels were never needed and we didn't want to add a label to every single namespace in the cluster.